docs/tutorial/ecommerce-tutorial/index.md

```diff
@@ -89,51 +95,74 @@ You can find an implementation of these examples [on GitHub](https://github.com/
 
 If you're selling a single product, like an eBook for example, you can create a single button that will perform a redirect to the Stripe Checkout page:
 
-#### Create a product and SKU
+#### Create products and prices
 
-To sell your products, you need to create them in your Stripe account using the [Stripe Dashboard](https://dashboard.stripe.com/products) or the [Stripe API](https://stripe.com/docs/api/products/create). This is required for Stripe to validate that the request coming from the frontend is legitimate and to charge the correct amount for the selected product/SKU. Stripe requires every SKU used with Stripe Checkout to have a name: be sure to add one to all of your SKUs.
+To sell your products, you need to create them in your Stripe account using the [Stripe Dashboard](https://dashboard.stripe.com/products) or the [Stripe API](https://stripe.com/docs/api/products/create). This is required for Stripe to validate that the request coming from the frontend is legitimate and to charge the correct amount for the selected product/price.
 
-You will need to create both test and live product SKUs separately in the Stripe Dashboard. **Make sure you toggle to "Viewing test data", then create your products for local development.**
+You will need to create both test and live products separately in the Stripe Dashboard. **Make sure you toggle to "Viewing test data", then create your products for local development.**
 
 #### Create a checkout component that loads Stripe.js and redirects to the checkout
 
-Create a new file at `src/components/checkout.js`. Your `checkout.js` file should look like this, with `loadStripe` updated with your API key and `lineItems` with one of your product IDs from the Stripe dashboard:
+Create a new file at `src/components/checkout.js`. Your `checkout.js` file should look like this. Make sure to add your publishable key in the `loadStripe` method and replace the price ID in the `lineItems` with one of your price IDs from the Stripe dashboard:
 
 ```jsx:title=src/components/checkout.js
-import React from "react"
+import React, { useState } from "react"
 import { loadStripe } from "@stripe/stripe-js"
```

Comment:
I think you might have to clean your stripe.com cookies (or it will update when they expire): https://github.com/stripe/react-stripe-elements/issues/445#issuecomment-634166203